frontend/src/metabase-lib/types.ts

```diff
@@ -258,39 +259,9 @@ export type OrderByClauseDisplayInfo = ClauseDisplayInfo & {
   direction: OrderByDirection;
 };
 
-export type ExpressionOperator =
```

Comment:
This list is always forgotten when adding clauses to `config` so I derived it's type from the defined clauses in config.